### ISSUE TITLE
Hello .Thanks for your work . Would you mind provide the detail of the maskrcnn ?

### ISSUE THREAD

**Issue by baibizhe** [Schuture/Meta-USCL#2]
Hello.Thanks for your wonderful work.
Would you mind providing the implementation detail of the segmentation model,which is Maskrcnn? Ex , the anchor_generator detail, the loss function has been used ... etc.

Best regards,
Bizhe

**Comment by Schuture** [comment]
No problem. We have uploaded our codes for fine-tuning Mask-RCNN to this repo. Instruction is also added to README.

**Comment by baibizhe** [comment]
Thanks  a lot ! And really appreciate your help.
